src/auto-evo/AutoEvoRun.cs

```diff
@@ -272,9 +276,13 @@ public string MakeSummaryOfExternalEffects()
         foreach (var entry in combinedExternalEffects)
         {
             builder.Append(entry.Key.Item1.FormattedName);
-            builder.Append(" population changed by ");
+            builder.Append(" ");
+            builder.Append(TranslationServer.Translate("AUTOEVO_POPULATION_CHANGE"));
+            builder.Append(" ");
             builder.Append(entry.Value);
-            builder.Append(" because of: ");
+            builder.Append(" ");
+            builder.Append(TranslationServer.Translate("AUTOEVO_BECAUSE"));
+            builder.Append(" ");
```

Comment:
I think that this entire part should be a single string that is formatted to allow translations to change the order of the numbers and words as that is probably required for some languages to sound natural.